doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1090 @@
+      * :kconfig:option:`CONFIG_ZIGBEE_SHELL_ZCL_CMD_TIMEOUT` for timing out ZCL cmd commands.
+
+    * Updated:
+
+      * :ref:`lib_zigbee_shell` structure to make it an independent library.
+      * File names ``zigbee_cli*`` and changed it to ``zigbee_shell*``.
+      * Function names ``zigbee_cli*`` and changed it to ``zigbee_shell*``.
+      * ``bdb factory_reset`` command.
+        Now the command checks if the ZBOSS stack is started before performing the factory reset.
+      * ``zcl cmd`` shell command extended to allow sending groupcasts.
+      *  ``zdo`` shell commands extended to allow binding to a group address.
+      * Internal context manager structures.
+
+    * Fixed:
+
+      * The issue where the ``zcl cmd`` shell command was using the incorrect index of a context manager entry during cleanup after the command was sent.
+
+  * :ref:`lib_zigbee_zcl_scenes` library:
+
+    * Updated the library, so that it is allowed to store empty scenes.
+
+  * :ref:`lib_zigbee_osif` library:
+
+    * Updated:
+
+      * Crypto library used for performing software AES ecryption.
+        Now, the :ref:`nrfxlib:nrf_oberon_readme` is used instead of the Tinycrypt library.
+      * Optimize calling ZBOSS API in |NCS| platform.
+        If the ZBOSS API is called in the ZBOSS thread context, processing by the workqueue is now skipped.
+
+  * :ref:`lib_zigbee_fota` library:
+
+    * Added:
+
+      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` Kconfig option to configure the interval between queries for the Zigbee FOTA server.
+      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN` Kconfig option to configure the interval between queries for the available Zigbee FOTA images.
+      * Support for the combined application and network core updates for the nRF5340 SoC.
+
+    * Updated:
+
+      * Download logic to use the :ref:`lib_dfu_multi_image` library API and image structure.
+      * The image generation script by introducing the sub-element structure inside the Zigbee OTA image.
+        Enable :kconfig:option:`CONFIG_ZIGBEE_FOTA_GENERATE_LEGACY_IMAGE_TYPE` to generate images compatible with previous |NCS| releases.
+      * Default value of the :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_TYPE` to 0x0141.
+      * Setting of the :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` for nRF5340 SoC to ensure integrity of the upgrade image.
+
+    * Removed:
+
+      * The :c:enum:`ZIGBEE_FOTA_EVT_ERASE_PENDING` and :c:enum:`ZIGBEE_FOTA_EVT_ERASE_DONE` events.
+
+  * Fixed:
+
+    * An issue where printing binding table containing group-binding entries results in corrupted output.
+    * An issue where Zigbee shell coordinator would not form a new network after the factory reset operation.
+
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
```

Comment:
I think that for crypto we can have the same style of  updates as the 1.9 release:

`* Updated:

  * :ref:`crypto`:

    * The Oberon library now  supports PKCS#7 padding with CBC  and 16 bytes IV with GCM mode in PSA APIs
    * Updated:

      * :ref:`nrfxlib:nrf_cc3xx_platform_readme` to v0.9.14.
      * :ref:`nrfxlib:nrf_cc3xx_mbedcrypto_readme` to v0.9.14.
      * :ref:`nrfxlib:nrf_oberon_readme` to v3.0.11.`

Reply:
Yes, this should suffice

Reply:
done

Reply:
In addition, there are mainly these highlights:
nrf_dm / nordic distance toolbox:
- now supports nrf52832 and nrf5340.
softdevice controller:
- added support for connectionless angle of arrival (AoA) transmitter
- added support for peripheral-initiated feature exchange
- added a nak_count field into QoS Connection event reports.